### DISCUSSION TITLE
Not Receiving Login Email for Admin Account on AWS

### DISCUSSION THREAD

**Discussion by mechaneyes** [mozilla/hubs#3166] [Hubs Cloud Questions] [answered]
Hello all,

I'm unable to log into my AWS hosted HC install. What's worse is after starting with a fresh clone of HC I can't login to the local custom client, as emails aren't being sent/received.

From SES Email Addresses screen I'm able to send and receive test emails.

This all started when I tried to `npm run deploy` while the webpack-dev-server was running. This couldn't have broken it this badly though, right?

Thanks for your help!
Ray

**Comment by mechaneyes** [comment]
Now that I'm remembering what steps I've tried, I'm realizing I deleted the CNAME records expecting them to be recreated. Is there a document I can check that details the process of getting this setup?

**Comment by robinkwilson** [comment]
`npm run deploy` shouldn't have broken the email system. I'd run `npm run undeploy` to remove the custom client, to be safe, then try logging in with your admin email address. See if that fixes anything.

If not, can you go through the steps in the Hubs Docs to check your admin email address? https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#then-what-is-my-hub-stacks-admin-email-address

If you've deleted CNAMES you may need to delete the stack, then recreate it with the same parameters since the AWS system uses those CNAMES.

Let me know how it goes.

**Comment by mechaneyes** [comment, marked as answer]
Thanks for getting back, Robin,

Yesterday ... I took the stack offline and tried to create a new one. In
the setup it decided to roll back the stack. Apparently because I'd reached
my quota of ACM certificates.

Am I right in thinking that I won't be able to create and use a new stack
until the quota is increased? I've submitted a ticket asking for the
increase.


Running `npm run undeploy` didn't help, nor did the troubleshooting steps
in the docs.

After undeploy I'm getting this error:

WebSocket connection to 'wss://
clever-druid.stereoh-dev.de/socket/websocket?vsn=2.0.0' failed: Error
during WebSocket handshake: Unexpected response code: 403

That makes me think of the CNAME issue again. Thoughts?


Thanks again for your help!

**Reply by robinkwilson** [reply]
If you took the stack offline / or deleted the stack, your `npm run undeploy` isn't going to work because nothing is running on the other end! Makes sense. 

Yes, wait 1-3 days for the certificate increase to happen then deploy again (I also had to wait on this too my first deploy).

**Reply by robinkwilson** [reply]
[Temporarily marked as answered] Unmark this as "answered", when you deploy again. We'll continue then.

**Reply by mechaneyes** [reply]
I wiped everything out and started fresh. All is working again. thanks!

**Comment by mechaneyes** [comment]
OK. Cool. Much appreciated, Robin.
